core/src/main/scala/quasar/plugin/postgres/datasource/PostgresDatasourceModule.scala

```diff
@@ -164,18 +168,20 @@ object PostgresDatasourceModule extends DatasourceModule with Logging {
       kind, c, new RuntimeException("Connection is invalid."))
 
   private def hikariTransactor[F[_]: Async: ContextShift](
-      connUri: URI,
-      connPoolSize: Int,
+      config: Config,
+      properties: Properties,
       connectPool: ExecutionContext,
       xaBlocker: Blocker)
       : Resource[F, HikariTransactor[F]] = {
 
     HikariTransactor.initial[F](connectPool, xaBlocker) evalMap { xa =>
       xa.configure { ds =>
         Sync[F] delay {
-          ds.setJdbcUrl(s"jdbc:$connUri")
+          ds.setJdbcUrl(s"jdbc:${config.connectionUri}")
           ds.setDriverClassName(PostgresDriverFqcn)
-          ds.setMaximumPoolSize(connPoolSize)
+          ds.setMaximumPoolSize(config.poolSize)
+          ds.setAutoCommit(false)
```

Comment:
What's the idea behind setting auto commit to false?

Reply:
`defaultRowFetchSize` might not work w/o it in some situation if I understand correctly